Editor/Window/GameLiftRequestAdapter.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class GameLiftRequestAdapter
+    {
+        private GameLiftPlugin _gameLiftConfig;
+        private string _fleetName;
+        private string _fleetId;
+        public const string FleetLocation = "custom-location-1";
+        private const string FleetDescription = "Deployed by the Amazon GameLift Plug-in for Unity.";
+        private VisualElement _container;
+
+        public GameLiftRequestAdapter(GameLiftPlugin gameLiftConfig)
+        {
+            _gameLiftConfig = gameLiftConfig;
```

Comment:
Can we instead do something like

```
_gameLiftWrapper = gameLiftConfig.GameLiftWrapper
_gameLiftCoreApi = gameLiftConfig.GameLiftCoreApi
```

and reference these everywhere? Makes the number of references lesser, and cleans up the code as well. 
```

Reply:
refactored